gson/src/test/java/com/google/gson/FieldNamingPolicyTest.java

```diff
@@ -0,0 +1,78 @@
+package com.google.gson;
+
+import org.junit.Test;
+import org.junit.runners.Parameterized.Parameters;
+
+import static org.junit.Assert.assertEquals;
+
+public class FieldNamingPolicyTest {
+	
+
+	@Test
+	public void testUpperCaseFirstLetterNonUpperCaseLetter() {
```

Comment:
Just installed Mars....  auto formatting wasn't set.  I pulled the formatted in from lib.  that will be in, in the next commit.